### PR TITLE
Upgraded to Swift 3 syntax

### PR DESCRIPTION
**Pull request by nfgrilo**
# READY FOR REVIEW

Upgrades to Swift 3 syntax (#6)

### CODE REVIEW

File: swift.mustache [hunk 1]
```diff
@@ -45,35 +45,35 @@ class MyRequestController {
         {{/body.url_encoded_body}}
         ]
         let bodyString = bodyParameters.queryParameters
-        request.HTTPBody = bodyString.dataUsingEncoding(NSUTF8StringEncoding, allowLossyConversion: true)
+        request.httpBody = bodyString.data(using: String.Encoding.utf8, allowLossyConversion: true)
```
Comment (mittsh) [inline swift.mustache]: just `.utf8` isn't enough?

File: swift.mustache [hunk 2]
```diff
@@ -102,25 +102,25 @@ extension Dictionary : URLQueryParameterStringConvertible {
     var queryParameters: String {
         var parts: [String] = []
         for (key, value) in self {
-            let part = NSString(format: "%@=%@",
-                String(key).stringByAddingPercentEncodingWithAllowedCharacters(NSCharacterSet.URLQueryAllowedCharacterSet())!,
-                String(value).stringByAddingPercentEncodingWithAllowedCharacters(NSCharacterSet.URLQueryAllowedCharacterSet())!)
+            let part = String(format: "%@=%@",
+                String(describing: key).addingPercentEncoding(withAllowedCharacters: CharacterSet.urlQueryAllowed)!,
+                String(describing: value).addingPercentEncoding(withAllowedCharacters: CharacterSet.urlQueryAllowed)!)
```
Comment (mittsh) [inline swift.mustache]: Just `.urlQueryAllowed`?

**Review by mittsh — COMMENTED** [review]
Great! Just wondering if we could simply the access to enums.

It's funny to see how basically your PR is to do a replace `/^NS//` 😉

File: swift.mustache (hunk 1 above)
Comment (nfgrilo) [reply swift.mustache]: of course! thanks!

File: swift.mustache (hunk 2 above)
Comment (nfgrilo) [reply swift.mustache]: yes, same here! thanks

**Review by nfgrilo — COMMENTED** [review]
Fixed!

**Review by mittsh — APPROVED** [review]
Wonderful! Swift 3 FTW

**Pull request merged**